presentations.md

```diff
@@ -0,0 +1,29 @@
+# Presentations
+
+Every **1st Saturday** of the month from **3:30 PM EAT** @nairobilug hosts a Meetup with presentations. This document seeks to guide a speaker by giving them the necessary resources they need if they want to contribute a presentations.
+Video recordings of previous meetups are in a [YouTube Playlist](https://youtube.com/playlist?list=PLWYgxwb8YFV1OS4_kWef90dYJvm730x0I)  and text based write-ups are in a [GitHub Repo](https://github.com/nairobilug/presentations).
+
+## Step One
+Have a relevant **Linux** topic you want to discuss about.
+
+**N.B** Look out for duplicates from previous presentations. Maybe you want to build on top of previous submissions.
+
+## Step Two
+Announce your interest to the group by atleast the Tuesday leading to the presentation day (Saturday).
+
+## Step Three
+Video setup .....
+
+## Step Four
+Write-Up Formats and Template located @ https://github.com/nairobilug/presentations ......
+
+## Important Contacts
+- Raymond Wanyoike (RayWan)
+- Jason Rogena
+- Canine Mwenja (rookie101)
+- Boniface Munyoki (BonifaceKilz)
+
+### To-DO/ Discuss
+- Code of Conduct for group
```

Comment:
check here for the code of conduct suggested: https://github.com/nairobilug/nairobilug.or.ke/pull/183